packages/text-field/index.tsx

```diff
@@ -294,8 +346,9 @@ class TextField extends React.Component {
       <FloatingLabel
         className={floatingLabelClassName}
         float={this.state.labelIsFloated}
-        handleWidthChange={
-          (initialLabelWidth) => this.setState({initialLabelWidth})}
+        handleWidthChange={(initialLabelWidth) =>
```

Comment:
do we need to type `initialLabelWidth`?

Reply:
It is implied by the FloatingLabel definition.